src/client/pages/Varen/Varen.tsx

```diff
@@ -66,14 +66,20 @@ export function Varen() {
   ) : null;
 
   const vergunningenTables = Object.entries(tableConfig).map(
-    ([kind, { title, displayProps, filter, sort }]) => {
+    ([kind, { title, displayProps, filter, sort, listPageRoute }]) => {
```

Comment:
Nit: Ik zou zelf
```suggestion
    ([kind, conf]) => {
```
of
```suggestion
    ([kind, conf]) => {
    const { title, displayProps, filter, sort, listPageRoute } = conf;
```
doen. Ik vind zoveel haakjes op 1 lijn best moeilijk te lezen. En als je alles `conf.` maakt het ook duidelijker vind ik.